nixos/modules/services/monitoring/prometheus/exporters/pve.nix

```diff
@@ -100,6 +100,7 @@ in
   };
   serviceOpts = {
     serviceConfig = {
+      DynamicUser = cfg.environmentFile == null;
```

Comment:
Thanks for the review. i hope that the changes do fit what your suggestion looked for. 
I had to change the config file then to point to the path instead of the string then. I have retested both scenarios. The only downside on that is that systemd loads now the credential even when its now needed as its world readable in the case that its the empty config.

Reply:
TBH, it's fine if the empty config file is world readable since, unfortunately, the exporter requires a config file to be provided, even if it's empty.

Reply:
Sure I just tried to point out, that there is now extra work done in the way your requested it, as now systems does copy a world readable file to /run/credentials and make it a secret. :) That was the reason I gated the LoadCredential behind having a non empty config file. Nevertheless the way you suggested it, it's more clear to read and less code. Thanks for the review & approval.